tests/test_bills.py

```diff
@@ -14,14 +14,13 @@
 from lametro.models import LAMetroBill
 from lametro.utils import format_full_text, parse_subject
 
-
 # This collection of tests checks the functionality of Bill-specific views, helper functions, and relations.
 def test_bill_url(client, bill):
     '''
     This test checks that the bill detail view returns a successful response.
     '''
     bill = bill.build()
-    url = reverse('bill_detail', kwargs={'slug': bill.slug})
+    url = reverse('lametro:bill_detail', kwargs={'slug': bill.slug})
```

Comment:
This test is now failing; I took a look at the test fixture, but the `restrict_view` property is set to `False`, so I'm not sure why we're getting a 404 here. Haven't dug deep into this yet, though.

Reply:
I'd recommend checking out [the filter logic in the Metro bill manager](https://github.com/datamade/la-metro-councilmatic/blob/24cb5635b6958ade2bd9ded2c041dc2b64c1e4ee/lametro/models.py#L61). You'll have to make sure the bill meets at least one of the criteria for being shown. I'd pick the easiest one. 😇